src/spac/spatial_analysis.py

```diff
@@ -75,34 +80,49 @@ def Neighborhood_Enrichment_Analysis(
                 adata,
                 new_annotation_name,
                 ax,
-                return_matrix=False
+                return_matrix=False,
+                title=None,
+                seed=None
             ):
 
         # Calculate Neighborhood_Enrichment
         if return_matrix:
             matrix = sq.gr.nhood_enrichment(
                         adata,
                         copy=True,
+                        seed=seed,
                         cluster_key=new_annotation_name
                 )
 
             sq.gr.nhood_enrichment(
                         adata,
+                        seed=seed,
                         cluster_key=new_annotation_name
                 )
         else:
             sq.gr.nhood_enrichment(
                         adata,
+                        seed=seed,
                         cluster_key=new_annotation_name
                 )
 
-        # Plot Neighborhood_Enrichment
-        sq.pl.nhood_enrichment(
-                    adata,
-                    cluster_key=new_annotation_name,
-                    ax=ax,
-                    **kwargs
-            )
+        if title:
+            # Plot Neighborhood_Enrichment
+            sq.pl.nhood_enrichment(
+                        adata,
+                        cluster_key=new_annotation_name,
+                        title=title,

```

Comment:
@ruiheesi, can title be None so we make one call?

Reply:
Sure George, I will update this. Thanks.